client/src/components/TicketForm/TicketForm.jsx

```diff
@@ -14,7 +14,6 @@ import './TicketForm.css';
 class TicketForm extends Component {
 	constructor(props) {
 		super(props);
-		console.log(props);
```

Comment:
We could enable an ESLint rule that errors on this if you want?

Reply:
That would be good